packager/live_packager.cc

```diff
@@ -206,15 +264,18 @@ Status LivePackager::Package(const Segment& in, FullSegmentBuffer& out) {
     return size;
   };
 
-  shaka::PackagingParams params;
-  params.chunking_params.segment_duration_in_seconds =
+  shaka::PackagingParams packaging_params;
+  packaging_params.chunking_params.segment_duration_in_seconds =
       config_.segment_duration_sec;
 
+  EncryptionParams& encryption_params = packaging_params.encryption_params;
+  internal_->segment_manager->InitializeEncryption(config_, encryption_params);
```

Comment:
could you please add a comment that this call will  fill data inside **encryption_params**
at first look it seems like encryption_params is an in-argument to this function.